### PR TITLE
update to cosas 0.8

### PR DESCRIPTION
**Pull request by eparejatobes**

### CODE REVIEW

**Comment by eparejatobes** [comment]
I am not incredibly happy with how default values for options work now. I think it would be cleaner if this would be ops for a command, will probably do that. In that case type inference would make things cleaner.

**Comment by eparejatobes** [comment]
apart from that I only need checkForAll or something like that for a predicate on a klist

**Comment by eparejatobes** [comment]
everything green here. Missing

- [ ] checkforall as explained above
- [ ] move defaults to a typeclass for blast command

**Comment by eparejatobes** [comment]
tests pass with 0.8.0, merging

File: src/main/scala/api.scala [hunk 1]
```diff
@@ -127,9 +133,11 @@ case object api {
     val command: BC
   )(
     val outputRecord: OR
-  )(implicit
-    val validRecord: OR#Properties CheckForAll ValidOutputRecordFor[BC]
   )
+  // TODO fix this!
+  // (implicit
+  //   val validRecord: OR#Keys CheckForAll ValidOutputRecordFor[BC]
+  // )
```
Comment (laughedelic) [inline src/main/scala/api.scala]: If I understand right, this is commented out just because in cosas-0.8 there were no `all`/`any` for predicates. Right?

Comment (eparejatobes) [inline src/main/scala/api.scala]: spot on

Comment (laughedelic) [inline src/main/scala/api.scala]: So should we release cosas-0.9 and update everything to it?

**Pull request merged**